Atomic/sign.py

```diff
@@ -0,0 +1,89 @@
+from . import util
+from . import Atomic
+import os
+import tempfile
+from .atomic import AtomicError
+from .client import ids_same, add_shaw
+
+
+class Sign(Atomic):
+    def sign(self):
+        # TODO
+        # If an alternate sign key is provided, should we verify that it is valid?
```

Comment:
Only that the path provided is actually a key.  In looking at this later today, I dont think this is really a concern.  The error message gpg gives is quite clear and I simply expose that from stderr.